### DISCUSSION TITLE
Can't see announcement banner content again after closing announcement banner

### DISCUSSION THREAD

**Discussion by mcbouslog** [zooniverse/front-end-monorepo#6630] [Feature Requests]
## Package
- lib-classifier

## Is your feature request related to a problem? Please describe.
As a user, I'd like to be able to see the Announcement Banner content again after closing/dismissing it.

## Describe the solution you'd like
Some way to see the Announcement Banner content again after closing/dismissing it.

For reference - PFE persists the Announcement Banner, doesn't allow it to be closed or dismissed.

**Comment by eatyourgreens** [comment]
See #594. The banner is supposed to appear again when you start a new session eg. open a new tab. I’m not sure that’s obvious though. How on earth would a volunteer know that?

**Comment by eatyourgreens** [comment]
See also #1176.

**Comment by goplayoutside3** [comment]
@seanmiller26 I converted this legacy Issue into a Discussion with the label "feature request". The discussion needs a concluding design and UX decision before it can be a helpful Issue.
